doc/markdownlinkconverter/markdownlinkconverter.c

```diff
@@ -60,6 +64,7 @@ struct transitionTitle { int t[10][4]; };
  */
 
 const int linkStart = 0;
+int lineCount = 0;
```

Comment:
If possible this should be avoided. For this kind of tools its ok, but it is definitely a code smell.